src/adera/ShipResources.h

```diff
@@ -198,4 +204,9 @@ inline MachineContainer& MachineContainer::operator=(MachineContainer&& move) no
     return *this;
 }
 
+struct ACompContainerShape
+{
+    osp::phys::ECollisionShape m_shape{osp::phys::ECollisionShape::NONE};
```

Comment:
It sounds like a good idea to separate collision shapes from inertia-only shapes, instead of having this specifically made for containers.